### ISSUE TITLE
scAtlas ontology

### ISSUE THREAD

**Issue by zoependlington** [EBISPOT/efo#159]
> Hi all,
>
> Good news! Most of the ontology terms that we are using in single cell Atlas are already in EFO. Find attached the terms that we would need to be added to EFO before releasing the single cell Atlas portal. We are finishing the analysis of another set of experiments and I will check the ontology annotations for this second set once our analysis is done. I will let you know if we need to add more terms to EFO from this second set of datasets.
> 
> Best wishes,
> 
> Laura

Terms can be found here: https://docs.google.com/spreadsheets/d/13vRnF1qYkvVpSp9OX2OndJLJog34MdQJfxh1Y9qLDag/edit?usp=sharing

**Comment by zoependlington** [comment]
A few notes to self:
- All terms have been added apart from 1: "pupal stage". Currently in EFO we have "pupa", an UBERON term (UBERON:0003143). After discussing this, we have decided that the two terms are different. The current term, pupa, in EFO, is a child of both animal component and developmental stage. We both feel this term reflects the animal component while the suggested term to import is more developmental stage. The proposal is to edit pupa, removing it as a child of developmental stage, and importing pupal stage as a child of developmental stage.
     - Will this impact anyone? Do we need to highlight this change? Is this split and import the correct path?
- Secondly, obstructive sleep apnea syndrome has been imported and merged with the existing EFO term obstructive sleep apnea including all x-refs, definitions etc.
